agents/check_mk_agent.aix

```diff
@@ -124,7 +124,9 @@ function run_cached {
         if (( $MTIME < $MAXAGE )) ; then
             USE_CACHEFILE=1
         fi
-        CACHE_INFO="cached($MTIME,$MAXAGE)"
+        # get file time for cache
+        FTIME=$(perl -le 'print((stat shift)[9])' $CACHEFILE)
```

Comment:
For the calculation of MTIME we use "/usr/bin/perl ...".
Please adapt your perl call or fix this calculation, too

Reply:
Perl call is ok.